en/tips/command-line.md

```diff
@@ -0,0 +1,77 @@
+
+| Command  | Description  |
+|---|---|
+| mautic:assets:generate  |  Combines and minifies asset files from each bundle into single production files |
+| mautic:broadcasts:send  | Process contacts pending to receive a channel broadcast.  |
+| mautic:campaigns:execute  | Execute specific scheduled events.  |
+| mautic:campaigns:messagequeue  | Process sending of messages queue.  |
+| mautic:campaigns:messages   | Process sending of messages queue.  |
+| mautic:campaigns:rebuild   | Rebuild campaigns based on contact segments.  |
+| mautic:campaigns:trigger  | Trigger timed events for published campaigns.  |
+| mautic:campaigns:update  | Rebuild campaigns based on contact segments.  |
+| mautic:campaigns:validate  | Validate if a contact has been inactive for a decision and execute events if so.  |
+| mautic:citrix:sync  | Synchronizes registrant information from Citrix products  |
+| mautic:contacts:deduplicate  | Merge contacts based on same unique identifiers  |
+| mautic:email:fetch  | Fetch and process monitored email.  |
+| mautic:emails:send   | Processes SwiftMail's mail queue  |
+| mautic:import  | Imports data to Mautic  |
+| mautic:install:data   | Installs Mautic with sample data  |
+| mautic:integration:fetchleads  | Fetch leads from integration.  |
+| mautic:integration:pipedrive:fetch  | Pulls the data from Pipedrive and sends it to Mautic  |
+| mautic:integration:pipedrive:push  | Pushes the data from Mautic to Pipedrive  |
+| mautic:integration:pushactivity  | Push lead activity to integration.  |
+| mautic:integration:pushleadactivity  | Push lead activity to integration.  |
+| mautic:integration:synccontacts  | Fetch leads from integration.  |
+| mautic:iplookup:download  | Fetch remote datastores for IP lookup services that leverage local lookups  |
+| mautic:maintenance:cleanup  | Updates the Mautic application  |
+| mautic:messages:send  | Process sending of messages queue.  |
+| mautic:migrations:generate  | Generate a blank migration class.  |
+| mautic:plugins:install  | Installs, updates, enable and/or disable plugins.  |
+| mautic:plugins:reload  | Installs, updates, enable and/or disable plugins.   |
+| mautic:plugins:update  | Installs, updates, enable and/or disable plugins.  |
+| mautic:queue:process   | Process queues  |
+| mautic:reports:scheduler  | Processes scheduler for report's export  |
+| mautic:segments:check-builders  | Compare output of query builders for given segments  |
+| mautic:segments:rebuild  | Update contacts in smart segments based on new contact data.  |
+| mautic:segments:update   | Update contacts in smart segments based on new contact data.  |
+| mautic:social:monitoring  | Looks at the records of monitors and iterates through them.   |
+| mautic:theme:json-config   | Converts theme config to JSON from PHP  |
+| mautic:transifex:pull  | Fetches translations for Mautic from Transifex  |
+| mautic:transifex:push   | Pushes Mautic translation resources to Transifex  |
```

Comment:
I meant these transifex commands. I don't even know what the translation commands do 😄

Reply:
I just removed those two and the others relating to translation!